projects/lemmy/services/lemmy/examples/basic.nix

```diff
@@ -0,0 +1,20 @@
+{ ... }:
+
+{
+  services.lemmy = {
+    enable = true;
+    ui.port = 3000;
+    database.createLocally = true;
+    settings = {
+      hostname = "http://hunter2lemmy.com";
+      port = 8000;
+      setup = {
+        admin_username = "AzureDiamond";
+        site_name = "Lemmy FTW";
+        admin_email = "AzureDiamond@hunter2.com";
+      };
+    };
```

Comment:
Can we perhaps make this a bit more generic? like `lemmylocal` or something.

Reply:
Sure thing lol. I was referencing this meme: https://knowyourmeme.com/memes/hunter2